website/docs/api-loss.md

```diff
@@ -57,8 +57,18 @@ Calculate the scalar loss. Typically returns a float.
 
 <inline-list>
 
+A flexible implementation of the common categorical cross-entropy loss that
+works on various data types. The **Guesses** should represent probabilities and
+are expected to be in the range of <tt>[0, 1]</tt>. They can both represent
+exclusive classes from multi-class cross-entropy (generally coming from a
+<tt>softmax</tt> layer) or could be classwise binary decisions for multi-label
+cross-entropy (<tt>sigmoid</tt> layer). The **Truths** are most commonly
+provided as labels in <tt>Ints1d</tt>, <tt>List[int]</tt> or <tt>List[str]</tt>
+format. Alternatively, users can provide a <tt>Floats2d</tt> which to encode
+label-confidence.
```

Comment:
In a text like this, I would format these differently:
```suggestion
A flexible implementation of the common categorical cross-entropy loss that
works on various data types. The `guesses` should represent probabilities and
are expected to be in the range of <tt>[0, 1]</tt>. They can both represent
exclusive classes from multi-class cross-entropy (generally coming from a
<tt>softmax</tt> layer) or could be classwise binary decisions for multi-label
cross-entropy (<tt>sigmoid</tt> layer). The `truths` are most commonly
provided as labels in <tt>Ints1d</tt>, <tt>List[int]</tt> or <tt>List[str]</tt>
format. Alternatively, users can provide a <tt>Floats2d</tt> which to encode
label-confidence.
```
I'm also unsure about the rendering of the `<tt>` tags within the text. Can't we just use backticks for those? At any rate, they don't seem to be appropriate for marking up words like `softmax`, cf https://github.com/explosion/thinc/tree/master/website#tt-type-annotation.

Also, this text should be moved outside of the `<inline-list>` scope.

Reply:
I'll swap all of them to backticks, makes more sense I agree.